linux/just_git_airgap_repo.bsh

```diff
@@ -1062,96 +1204,56 @@ function relocate_git_defaultify()
         echo
       fi
 
-      JUST_IGNORE_EXIT_CODES=3
+      JUST_IGNORE_EXIT_CODES=123
 
       local check_remote
       local remote_name
       local prep_dir
       local branch
       local output_dir
-      # In some versions of bash, these variables must explicitly be set to null
-      unset check_remote
-      unset remote_name
-      unset prep_dir
-      unset branch
-      unset output_dir
-
-      # REVIEW Switch to command_tools:parse_args
-      # RE Doesn't support help or die on unprocessed flags/positional arguments,
-      # so I'd have to have a while loop like this to deal with those anyway;
-      # Also have to handle the --no- variant manually
-      while (( $# )); do
-        # Convert arguments of the form -f=val to -f val and --flag=val to --flag val
-        # https://medium.com/@binaryphile/nice-one-elegant-in-its-simplicity-eee6b433bc90
-        if [[ $1 == -?=* ]] || [[ $1 == --*=* ]]; then
-          set -- "${1%%=*}" "${1#*=}" "${@:2}"
+
+      local parsed_args
+      local usage_help=0
+      # TODO Support --remote-url (but disallow also setting --check-remote)
+      parse_args parsed_args --help usage_help -h usage_help \
+          -b branch: --branch branch: \
+          -c check_remote --check-remote check_remote --no-check-remote check_remote! \
+          -r remote_name: --remote-name remote_name: -p prep_dir: --prep-dir prep_dir: \
+          -o output_dir: --output-dir output_dir: -- ${@+"${@}"}
+      shift "${parsed_args}"
+      if [ $# -gt 0 ]; then
+        if [[ "$1" = "-"* ]]; then
+          echo "Unknown flag: $1"
+        else
+          echo "Positional arguments are not allowed: $1"
         fi
```

Comment:
You are checking to see if there are any unparsed args here.

Just keep in mind that "chained just targets" would have resulted in unparsed flags here, so you are indirectly saying you can't have chained a target with this target. That is allowed, I just want you to be aware what this is really doing.